data.json

```diff
@@ -14,7 +14,8 @@
   "Facebook": {
     "url": "https://www.facebook.com/{}",
     "urlMain": "https://www.facebook.com/",
-    "errorType": "status_code"
+    "errorType": "status_code",
+    "regexCheck": "^([a-zA-Z0-9.]){5,}$"
```

Comment:
We should remove this file from this PR, as it kinda messes up `git blame`.

Do you need help with this? I'm happy to walk you through the process!

Reply:
Yes, it would be very helpful. Can I do it from here?

Reply:
Ok, I'm waiting.